integrationtests/server-integration/third-party-server/src/test/resources/mariadb.properties

```diff
@@ -0,0 +1,5 @@
+id.column.type=VARCHAR(255)
```

Comment:
I think we should store the properties for: DB2, Oracle, MySQL, and others that we are not running upstream in a downstream project.